src/boss/lambdastore/store.go

```diff
@@ -264,8 +296,8 @@ func (s *LambdaStore) removeFromRegistry(funcName string) error {
 
 	// Background deletion
 	go func() {
-		if err := os.Remove(trashPath); err != nil {
-			log.Printf("warning: failed to remove %s from trash: %v", trashPath, err)
+		if err := s.bucket.Delete(context.Background(), funcName+".tar.gz"); err != nil {
```

Comment:
It seems like we hardcode .tar.gz in a lot of places.  Might be a pain later if we use a different format.  At the least, how about a const for now?